web/views/bootstrap_view.ex

```diff
@@ -6,7 +6,7 @@ defmodule Cog.V1.BootstrapView do
   end
 
   def render("bootstrapped.json", _) do
-    %{bootstrap: %{error: "Already bootstrapped"}}
+    %{bootstrap: %{errors: "Already bootstrapped"}}
```

Comment:
This might cause some breakage in cogctl, but it'd be nice to swap the nesting of this error map to be `%{errors: %{bootstrap: "Already bootstrapped"}}`